src/SDL_ttf.c

```diff
@@ -883,11 +883,13 @@ static SDL_INLINE void BG_64(const TTF_Image *image, Uint8 *destination, Sint32
     Uint64       *dst    = (Uint64 *)destination;
     Uint32        width  = image->width / 8;
     Uint32        height = image->rows;
-
+    Uint64        tmp;
+    
     while (height--) {
         /* *INDENT-OFF* */
         DUFFS_LOOP4(
-            *dst++ |= *src++;
+            memcpy(&tmp, src++, sizeof(tmp));
```

Comment:
Is `memcpy` ok? Or do we want `SDL_memcpy`?

This creates a dependency on libc, or don't we care about this in the satellite libs?

Reply:
We want memcpy so that compiler can remove it if possible.
I think this code is almost never compiled because there are neon and sse routines also.